dataRetriever/factory/resolverscontainer/shardResolversContainerFactoryCreator_test.go

```diff
@@ -0,0 +1,28 @@
+package resolverscontainer_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/multiversx/mx-chain-go/dataRetriever/factory/resolverscontainer"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewShardResolversContainerFactory(t *testing.T) {
```

Comment:
TestNewShardResolversContainerFactoryCreator ?

Reply:
Right, fixed it